src/app/charts/core/files/config/txt/seed-content.s3.yaml

```diff
@@ -20,3 +20,5 @@ buckets:
     # version:
     #   mfaDel: true
     #   enable: true
+  arkcase-config:
+    acl: "private"
```

Comment:
This is not necessary, since it's taken care of by the internal content store's initialization.